docs/docs/chitchat-faqs.mdx

```diff
@@ -145,6 +107,58 @@ nlu:
     - cheers
 ```
 
+### 2. Writing stories
```

Comment:
This entire page will be rewritten/restructured here: https://github.com/RasaHQ/rasa/pull/6373
Wouldn't recommend putting too much work into it 😃 (as well as all of the conversations pattern docs)